scripts/testnet/sovereignBridge/config/configs.cfg

```diff
@@ -1,54 +1,49 @@
-#============ OWNER ==============
+#============ SOVEREIGN PATHS ==============
+SOVEREIGN_DIRECTORY="~/sovereign"
+TXS_OUTFILE_DIRECTORY="${SOVEREIGN_DIRECTORY}/txsOutput"
+CONTRACTS_DIRECTORY="${SOVEREIGN_DIRECTORY}/contracts"
 
+#============ OWNER ==============
 WALLET="~/wallet.pem"
 
-#============ WASM FILES ==============
-
-ESDT_SAFE_WASM="~/contracts/esdt-safe.wasm"
-FEE_MARKET_WASM="~/contracts/fee-market.wasm"
-MULTISIG_VERIFIER_WASM="~/contracts/multisigverifier.wasm"
+#=========== NETWORK CONFIGURATION ===========
+PROXY=https://testnet-gateway.multiversx.com
+CHAIN_ID=T
+PROXY_SOVEREIGN=http://localhost:7950
+CHAIN_ID_SOVEREIGN=local-testnet
 
-#=========== CHAIN SPECIFIC CONSTANTS ===========
+#=========== NETWORK CONSTANTS ===========
 ESDT_SYSTEM_SC_ADDRESS=erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u
 ESDT_ISSUE_COST=50000000000000000
 
-PROXY=https://testnet-gateway.multiversx.com
-CHAIN_ID=T
+#============ WASM FILES ==============
+ESDT_SAFE_WASM="${CONTRACTS_DIRECTORY}/esdt-safe.wasm"
+FEE_MARKET_WASM="${CONTRACTS_DIRECTORY}/fee-market.wasm"
+HEADER_VERIFIER_WASM="${CONTRACTS_DIRECTORY}/header-verifier.wasm"
+
+#============ FEE MARKET DEPLOY ARGUMENTS ==============
 PRICE_AGGREGATOR_ADDRESS=erd1qqqqqqqqqqqqqpgqfsfcrxnlym0fjlnuvus2dmlz6ju9cpsfexks8wq4d4
 WEGLD_TOKEN_ID=WEGLD-a28c59
 USDC_TOKEN_ID=USDC-350c4e
 
-PROXY_SOVEREIGN=http://localhost:7950
-CHAIN_ID_SOVEREIGN=local-testnet
-
-#============ ESDT SAFE DEPLOY ARGUMENTS ==============
```

Comment:
So these are not needed anymore?

Reply:
Correct. We created another version of the contract without those arguments.